tasks/webpack.dev.js

```diff
@@ -3,7 +3,6 @@ const common = require('./webpack.common.js');
 
 module.exports = merge(common, {
   output: {
-    path: `${__dirname}/../js`,
```

Comment:
Removing path did not work me first I had to change path to: 
```
path: `${__dirname}`,
```
Then webpack-dev-server ran just fine. Even if I removed path altogether.